src/cascadia/TerminalSettingsModel/CascadiaSettings.h

```diff
@@ -139,6 +140,15 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         size_t _userProfileCount = 0;
     };
 
+    struct ActionArgFactory
+    {
+        ActionArgFactory() = default;
+
+        static winrt::hstring GetNameForAction(ShortcutAction action);
+        static Windows::Foundation::Collections::IMap<Model::ShortcutAction, winrt::hstring> AvailableShortcutActionsAndNames();
+        static Model::IActionArgs GetEmptyArgsForAction(Model::ShortcutAction shortcutAction);
+    };
+

```

Comment:
Curious. Why is this next to CascadiaSettings instead of ActionMap? I feel like it would make more sense closer to all the other actions-related objects.

Reply:
Moved to action map!